Game/asteroid_view_model.gd

```diff
@@ -4,8 +4,10 @@ extends Node
 signal building_on_cursor_changed()
 # subscribed to by asteroid
 signal update_ore_tilemaps()
+# subscribed to by asteroid
+signal update_buildings()
```

Comment:
Your comment sould explain why and why this updates, in addition to who might use it.